src/metaschema/oscal_assessment-common_metaschema.xml

```diff
@@ -1209,7 +1209,7 @@
                         </remarks>
                      </field>
                      <define-assembly name="related-response" max-occurs="unbounded">
-                        <formal-name>Action Reference</formal-name>
+                        <formal-name>Risk Response Reference</formal-name>
```

Comment:
Line 1172 indicates that the **Risk Log Entry**: "_Identifies the result of a task that occurred as part of managing an identified risk._" <-- this is equivalent to: "_Identifies an individual risk response that occurred as part of managing an identified risk_"
So maybe line 1213 should change from: "_Identifies an individual risk response that this log entry is for._" (please note the similarities with line 1172_equivalent) to saying this is a reference related to the log entry. Proposed description for the **Risk Response Reference**: "_Identifies a reference related to the individual risk response associated with this log entry._"

Reply:
Done.

Reply:
I believe the information is not clear. If my suggested text above was not appropriate, we need to find one that is. The existing <description>_Identifies an individual risk response that this log entry is for_.</description> is confusing to me. So a **Log Entry** logs a response (as part of managing risk), and it has a **Risk Response Reference**, that per <description>identifies the response the Log Entry document, so it identifies itself? I still think the description should read: <description>Identifies **a reference related to** the individual risk response associated with this log entry.</description>

Reply:
I used your text suggestion. Although I missed the leading 'I'. I'll fix this.